web/src/index.html

```diff
@@ -17,32 +17,42 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
     />
-    <link rel="stylesheet" href="css/gcping.css" />
-    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
```

Comment:
Along with this, we should be able to remove the CSS from L:12 once we are done with the material design changes.